ballerina/natives.bal

```diff
@@ -21,7 +21,7 @@ import ballerina/jballerina.java;
 # boolean isMatched = regex:matches("Ballerina is great", "Ba[a-z ]+");
 # ```
 #
-# + stringToMatch - The string to match the regex
+# + stringToMatch - The string to match the `regex`
```

Comment:
```suggestion
# + stringToMatch - The string to match the `regex`
```
Why do we need backticks here?

Reply:
@praneesha  Some have backticks but some haven't. So I added this to make it identical.
https://github.com/ballerina-platform/module-ballerina-regex/blob/main/ballerina/natives.bal#L32

Reply:
As per the offline discussion, Remove all the backticks from the `regex` words as this is a normal word.

Reply:
Ack. Thanks.